lib/msf/core/exploit/http/postfixadmin.rb

```diff
@@ -0,0 +1,26 @@
+module Msf
+  class Exploit
+    class Remote
+      module HTTP
+        module PostfixAdmin
+          require 'msf/core/exploit/http/postfixadmin/login'
+          require 'msf/core/exploit/http/postfixadmin/uris'
+
+          include Msf::Exploit::Remote::HttpClient
+          include Msf::Exploit::Remote::HTTP::PostfixAdmin::Login
+          include Msf::Exploit::Remote::HTTP::PostfixAdmin::URIs
+
+          def initialize(info = {})
+            super
+
+            register_options(
+              [
+                Msf::OptString.new('TARGETURI', [true, "The base path to the postfixadmin installation", "/"])
```

Comment:
does the default `targeturi` need to be overwritten just to add "postfixadmin installation"?